mlx/backend/metal/kernels/steel/host.h

```diff
@@ -1,5 +1,3 @@
 // Copyright © 2024 Apple Inc.
```

Comment:
Does this file have a purpose? It looks empty?

Reply:
I'll remove it for now - it will probably come back later but we can worry about that later